var/spack/repos/builtin/packages/py-picmistandard/package.py

```diff
@@ -49,14 +50,7 @@ class PyPicmistandard(PythonPackage):
         deprecated=True,
     )
 
-    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("python@3.8:", type=("build", "run"))
     depends_on("py-numpy@1.15:", type=("build", "run"))
     depends_on("py-scipy@1.5:", type=("build", "run"))
```

Comment:
I'm pretty sure `requires.txt` with:

```
numpy~=1.15
scipy~=1.5
```

means these should be:

```suggestion
    depends_on("py-numpy@1.15", type=("build", "run"))
    depends_on("py-scipy@1.5", type=("build", "run"))
```

See the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#ranges-versus-specific-versions.

Reply:
Oh, but that means:
```suggestion
    depends_on("py-numpy@1.15:1", type=("build", "run"))
    depends_on("py-scipy@1.5:1", type=("build", "run"))
```
does it?

Reply:
(I just recently relaxed it, but not yet in this release: https://github.com/picmi-standard/picmi/blob/master/PICMI_Python/requirements.txt)

Reply:
`@1.15` covers `1.15` through `1.15.z` while `@1.15:1` covers `1.15` through `1.y.z`, where `y` >= `15`.